sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -117,119 +142,88 @@ bool oclHandleInvalidWorkGroupSize(const device_impl &DeviceImpl,
          NDRDesc.GlobalSize[2] % NDRDesc.LocalSize[2] != 0);
     // Is the local size of the workgroup greater than the global range size in
     // any dimension? This is a sub-case of NonUniformWGs.
```

Comment:
```suggestion
    // any dimension?
```
This comment is confusing and the way I understand it, it doesn't seem to be correct.